src/hooks/use-chat-context.tsx

```diff
@@ -146,35 +160,27 @@ export const ChatProvider: React.FC< ChatProviderProps > = ( { children } ) => {
 		setThemesList( ( prev ) => ( { ...prev, [ selectedSite.id ]: themes } ) );
 	} );
 
-	const contextValue = useMemo( () => {
-		return {
-			numberOfSites,
-			themeList: selectedSite?.id ? themesList[ selectedSite.id ] || [] : [],
-			pluginList: selectedSite?.id ? pluginsList[ selectedSite.id ] || [] : [],
-			wpVersion,
-			phpVersion: selectedSite?.phpVersion ?? DEFAULT_PHP_VERSION,
-			currentURL: `http://localhost:${ sitePort }`,
-			themeName: themeDetails?.name,
-			isBlockTheme: themeDetails?.isBlockTheme,
-			availableEditors,
-			siteName: selectedSite?.name,
-			os: window.appGlobals.platform,
-		};
-	}, [
-		numberOfSites,
-		selectedSite?.id,
-		selectedSite?.phpVersion,
-		selectedSite?.name,
-		themesList,
-		pluginsList,
-		wpVersion,
-		sitePort,
-		themeDetails?.name,
-		themeDetails?.isBlockTheme,
-		availableEditors,
-	] );
-
-	return <ChatContext.Provider value={ contextValue }>{ children }</ChatContext.Provider>;
+	return (
+		<ChatContext.Provider
+			value={ {
+				availableEditors,
+				currentURL: `http://localhost:${ sitePort }`,
+				getChatInput,
+				isBlockTheme: themeDetails?.isBlockTheme,
+				numberOfSites,
+				os: window.appGlobals?.platform,
+				phpVersion: selectedSite?.phpVersion ?? DEFAULT_PHP_VERSION,
+				pluginList: selectedSite?.id ? pluginsList[ selectedSite.id ] || [] : [],
+				saveChatInput,
+				siteName: selectedSite?.name,
+				themeList: selectedSite?.id ? themesList[ selectedSite.id ] || [] : [],
+				themeName: themeDetails?.name,
+				wpVersion,
+			} }
```

Comment:
We don't strictly _need_ to get rid of the `useMemo` call, but I find it a code smell when values are unnecessarily memoized in React apps, and I couldn't see a reason why we'd need to memoize this one.